### PR TITLE
build: use correct version for multiarch image

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Pass correct version to multiacrh build targets.

Follow up on #517

### CODE REVIEW

File: controller.go [hunk 1]
```diff
@@ -286,7 +286,7 @@ func loadSettings() error {
 }
 
 func main() {
-	log.Infof("starting %s %s", os.Args[0], version)
+	log.Infof("Starting %s %s", os.Args[0], version)
```
Comment (AlexanderYastrebov) [inline controller.go:289]: Noticed in the v0.15.19 logs:
```
level=info msg="starting /kube-ingress-aws-controller v0.15.18-2-g0df1e50"
```

**Comment by mikkeloscar** [comment]
:+1:

**Comment by RomanZavodskikh** [comment]
:+1:

**Comment by AlexanderYastrebov** [comment]
DCO check is not working currently https://github.com/dcoapp/app/issues/211

**Pull request merged**